contracts/common/Registry.sol

```diff
@@ -118,6 +118,11 @@ contract Registry is Ownable {
     return rootToChildToken[_token] != address(0x0);
   }
 
+  function isTokenMappedAndIsErc721(address _token) public view returns (bool) {
+    require(isTokenMapped(_token), "TOKEN_NOT_MAPPED");
```

Comment:
please remove require condition.

Reply:
No, I want it to revert. Otherwise it will go to else block even if the token is not mapped.
```
if (registry.isTokenMappedAndIsErc721(_tokens[i])) {
        depositERC721ForUser(_tokens[i], _user, _amountOrTokens[i]);
      } else {
        depositERC20ForUser(_tokens[i], _user, _amountOrTokens[i]);
}
```

Reply:
aha! cool